### ISSUE TITLE
Remember me does not work with hotio radarr image

### ISSUE THREAD

**Issue by l00d3r** [hotio/radarr#26]
Every time I restart the radarr container and after that refresh my radarr's webpage I'm logged out even though I have checked the Remember me checkbox...

I'm running my radarr instance with the following config:

```
services:
  radarr:
    container_name: radarr
    image: ghcr.io/hotio/radarr:latest
    restart: unless-stopped
    logging:
      driver: loki
    ports:
      - 7878:7878
    environment:
      - PUID=1005
      - PGID=118
      - TZ=Europe/Amsterdam
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /data/containers/radarr:/config
      - /data:/data
```

I tested the exact same config with linuxserver's image and there is no such issue there, I tried finding what the difference with your and their image was, but unfortunately I did not find anything.

I have the same issue with hotios sonarr image and not with linuxservers image, so looks like something deeper than just radarrs issue.

I have no basepaths, proxies or certificates in my setup and am accessing directly over ip and port.

**Comment by l00d3r** [comment]
found what the fix is, hotio image does not cd into the application directory whereas linuxservers image does...

Changing this file https://github.com/hotio/radarr/blob/fb059fcf4fb96a232dd89e1722d08d9e0b1ab1bb/root/etc/s6-overlay/s6-rc.d/service-radarr/run#L6
from
`exec s6-setuidgid hotio "${APP_DIR}/bin/Radarr" --nobrowser --data="${CONFIG_DIR}"`
to
`exec cd "${APP_DIR}/bin" s6-setuidgid hotio "${APP_DIR}/bin/Radarr" --nobrowser --data="${CONFIG_DIR}"`

and restarting will fix the on restart session expiry

**Comment by l00d3r** [comment]
Found the root cause of [this](https://learn.microsoft.com/en-us/aspnet/core/security/data-protection/configuration/overview?view=aspnetcore-8.0#setapplicationname)

> By default, the Data Protection system isolates apps from one another based on their content root paths, even if they share the same physical key repository. This isolation prevents the apps from understanding each other's protected payloads.


This is the reason why it's happening on hotios containers, asp.net core or something in the 'arrs default content root to current workdir which is unique for every restart:

> [Info] Microsoft.Hosting.Lifetime: Content root path: /run/s6-rc:s6-rc-init:JiPpiG/servicedirs/service-radarr

**Comment by mrhotio** [comment]
Will fix tomorrow, good find

**Comment by mrhotio** [comment]
@l00d3r please test the new release

**Comment by l00d3r** [comment]
I can confirm it looks like its fixed in the latest version [release-5.11.0.9244](https://hub.docker.com/layers/hotio/radarr/release-5.11.0.9244/images/sha256-cd2ce24c7b54bfcf3c750487c81a5f54add1ead401496690969acc23ba7581cb?context=explore) 

Thanks!
